### DISCUSSION TITLE
Is it possible to determine the size of a 3Dsolid?

### DISCUSSION THREAD

**Discussion by Fernando-cad** [mozman/ezdxf#1173] [Q&A]
Hello, I am new to ezdxf, I am trying to solve a problem, but it does not work, maybe I do not understand something from the documentation or what I want to do by means of ezdxf is not possible at all.
The task is to get the dimensions (length, width, height) of solids (3DSolid). How do I know I need to use <class 'ezdxf.entities.acis.Solid3d'>? Could you direct me to the right path to solve my problem?
[board.zip](https://github.com/user-attachments/files/17155650/board.zip)

**Comment by mozman** [comment]
No
